src/Pages/RegisterForm/RegisterForm.tsx

```diff
@@ -0,0 +1,162 @@
+import React, { useState } from 'react';
+import { Button, Form, Input } from 'antd';
+import type { FormItemProps } from 'antd';
+import './style.scss';
+
+const MyFormItemContext = React.createContext<(string | number)[]>([]);
+
+interface MyFormItemGroupProps {
+  prefix: string | number | (string | number)[];
+}
+
+function toArr(
+  str: string | number | (string | number)[]
+): (string | number)[] {
+  return Array.isArray(str) ? str : [str];
+}
+
+const MyFormItemGroup: React.FC<
```

Comment:
i suggest to move this component to  a separate file so this file is not too crowded/ long

Reply:
Divided a component to separate parts.